swift/ql/lib/codeql/swift/generated/Callable.qll

```diff
@@ -9,7 +9,7 @@ module Generated {
   class Callable extends Synth::TCallable, Element {
     /**
      * Gets the self param, if it exists.
-     * This is taken from the "hidden" AST and should only be used to be overridden by classes.
+     * This includes nodes from the "hidden" AST and can be used to be overridden by subclasses.
```

Comment:
I'm not entirely clear what the second part of this sentence is saying (before or after the edit).

Reply:
I'm not sure if this is right but I've come to:
```
This includes nodes from the "hidden" AST. It can be overridden in subclasses to change the behaviour of both the Immediate and non-Immediate versions.
```

Reply:
nice suggestion, committed as 7b181a2de0f9afe1d1d8e865a9c91ba18fe1425d